inst/recorder/app.R

```diff
@@ -107,7 +108,12 @@ codeGenerators <- list(
   },
 
   snapshot = function(event) {
-    "app$snapshot()"
+    if (load_test){
+      "app$takeScreenshot()"
```

Comment:
I think this needs a filename? The `snapshot` method automatically adds a filename and increments a counter, so that they're named 001.png, 002.png, and so on. Do you want behavior like that here?

Also, there should be a space before `{`.